csharp/README.md

```diff
@@ -0,0 +1,19 @@
+# XenServer.NET usage examples
+
+## How to run the examples
+
+Open `XenSdkSample.sln` inside Visual Studio 2022 and compile the solution.
+
+The solution project is a console application expecting three parameters:
+
+```
+<host>     : an https URL pointing at the server
+<username> : a valid user on the server (e.g. root)
+<password> : the user's password
+```
+
+Run it as follows:
+
+```
+.\XenSdkSample.exe <host> <username> <password>
```

Comment:
Python example includes port in the host URL. If this is required here also, should it be mentioned in the parameter description ?
Python example also has dummy values for host port and username in the example command. Might be nice to have these here too ?

Reply:
The port is optional - will correct,